libs/common/src/constants/facility-type-mapped.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @description
```

Comment:
should this be here?

Reply:
I think this is the most semantic use of jsDocs but this is not an established pattern here so removing it everywhere